packages/fullstack/README.md

```diff
@@ -63,6 +63,81 @@ async fn get_meaning(of: String) -> Result<Option<u32>, ServerFnError> {
 }
 ```
 
+## Axum Integration
+
+If you have an existing Axum router or you need more control over the server, you can use the [`AxumRouterExt`](https://docs.rs/dioxus-fullstack/latest/dioxus_fullstack/prelude/trait.DioxusRouterExt.html) trait to integrate with your existing Axum router.
+
+First, make sure your `axum` dependency is optional and enabled by the server feature flag. Axum cannot be compiled to wasm, so if it is enabled by default, it will cause a compile error:
+
+```toml
+[dependencies]
+dioxus = { version = "*", features = ["fullstack"] }
+axum = { version = "0.7.0", optional = true }
+
+[features]
+server = ["dep:axum", "dioxus/server"]
+web = ["dioxus/web"]
+```
+
+Then we can set up dioxus with the axum server:
+
+```rust
+#![allow(non_snake_case)]
+use dioxus::prelude::*;
+
+// The entry point for the server
+#[cfg(feature = "server")]
+fn main() {
+    // Get the address the server should run on. If the CLI is running, the CLI proxies fullstack into the main address
+    // and we use the generated address the CLI gives us
+    let cli_args = dioxus_cli_config::RuntimeCLIArguments::from_cli();
+    let address = cli_args
+        .as_ref()
+        .map(|args| args.fullstack_address().address())
+        .unwrap_or_else(SocketAddr::from(([127, 0, 0, 1], 8080)));
```

Comment:
I am suggesting to add [this cli logic](https://github.com/DioxusLabs/dioxus/blob/main/packages/fullstack/src/launch.rs#L131-L138).

With [this changes](https://github.com/DioxusLabs/dioxus/issues/2933#issuecomment-2338105792), it will just cause friction as port 8080 will conflict with the asset server.

Reply:
port 8080 is a fallback if the CLI doesn't pass the port for the proxy